### DISCUSSION TITLE
Will the docs be updated for Yarp.ReverseProxy v2.1 and .Net 8?

### DISCUSSION THREAD

**Discussion by DefinitelyADev** [dotnet/yarp#2351] [General] [answered]
Will the docs be updated for Yarp.ReverseProxy v2.1 for any changes/migration to 2.1 and changes due to .Net 8?

**Comment by MihaZupan** [comment, marked as answer]
Thank you for reminding us, the docs should indeed be updated to talk about 2.1 instead of 2.0.
Moved this into a tracking issue: #2352
